tests/client/amo/helpers.js

```diff
@@ -12,6 +12,7 @@ export const fakeAddon = {
     id: 123,
     license: { name: 'tofulicense', url: 'http://license.com/' },
     version: '2.0.0',
+    files: [],
```

Comment:
I rebased master onto this and pushed it here. Rebased cleanly so I think git was smart enough to realise we did the same thing ^_^